OOInteraction/src/commands/CSceneHandlerItemTest.cpp

```diff
@@ -53,76 +56,133 @@ bool CSceneHandlerItemTest::canInterpret(Visualization::Item*, Visualization::It
 }
 
 Interaction::CommandResult* CSceneHandlerItemTest::execute(Visualization::Item*, Visualization::Item*,
-		const QStringList&, const std::unique_ptr<Visualization::Cursor>&)
+		const QStringList&, const std::unique_ptr<Visualization::Cursor>&) __attribute__ ((optnone))
```

Comment:
Please don't commit these kinds of attributes, just keep them for local testing.